#!/usr/bin/env python
PACKAGE = "livox_hikcamera_cal"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("concave_hull_alpha", double_t, 0, "x of box center", 0.25, 0.10, 0.40)

# gen.add("double_param", double_t, 0, "A double parameter", 1.0, 0.0, 5.0)
# gen.add("str_param", str_t, 0, "A string parameter", "hello")
# gen.add("bool_param", bool_t, 0, "A boolean parametBer", True)

exit(gen.generate(PACKAGE, PACKAGE, "CalibrationParam"))